addons/website_sale_slides/static/src/js/slides_course_quiz.js

```diff
@@ -17,23 +20,46 @@ sAnimations.registry.websiteSlidesQuizNoFullscreen.include({
     }
 });
 
-Quiz.include({
-    xmlDependencies: (Quiz.prototype.xmlDependencies || []).concat(
-        ["/website_sale_slides/static/src/xml/website_sale_slides_quiz.xml"]
-    ),
-    events: _.extend({}, Quiz.prototype.events || {}, {
-        'click .o_wslides_sale_js_join_course': '_onClickJoinSale',
-    }),
+CourseJoinWidget.include({
     /**
-     * @private
+     * @override
+     */
+    init: function () {
+        var def = this._super.apply(this, arguments);
```

Comment:
init is not a deferred
and does not return anything.
so you could simply do : 
```
init: function () {
    this._super.apply(this, arguments);
    doYourStuff;
}
```
Anyway, DOM modif can be done in template directly